src/main/java/games/strategy/triplea/ui/EditPanel.java

```diff
@@ -273,12 +273,14 @@ public void actionPerformed(final ActionEvent event) {
           return;
         }
         final Set<TechAdvance> advance = new HashSet<>();
+        // FIXME: This try-catch block appears to be guarding against a ClassCastException.
+        // It should be removed and the underlying problem fixed.
```

Comment:
After a second review, I actually think the `ClassCastException` isn't possible.  However, I need to research it further, so I added the FIXMEs to address in a follow-up PR.